scripts/typescript_checks.py

```diff
@@ -189,185 +106,60 @@
     'core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-admin-page.component.spec.ts',
     'core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-admin-page.component.ts',
     'core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-admin-page.import.ts',
+    'core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.constants.ajs.spec.ts',
```

Comment:
Please keep the files in this list alphabetized

Reply:
Done